p2p/transport/websocket/listener.go

```diff
@@ -124,13 +130,46 @@ func (l *listener) serve() {
 	}
 }
 
+// isProxyTrusted checks if the given IP address belongs to a trusted proxy
+func (l *listener) isProxyTrusted(remoteAddr string) bool {
+	if len(l.trustedProxies) == 0 {
+		// allow any
+		return true
```

Comment:
This seems surprising. I'd rather have an explicit match everything mask (i.e. 0.0.0.0/0)

Reply:
I agree. I removed allow any.